apiserver/HACluster.md

```diff
@@ -0,0 +1,215 @@
+# Creating HA cluster with API Server
+
+One of the issue for long-running Ray applications, for example, Ray Serve is that Ray Head node is a single
+point of failure, which means that if the Head node dies, complete cluster has to be restarted. Fortunately,
+KubeRay cluster provides an option to create
+[fault tolerance Ray cluster](https://docs.ray.io/en/master/cluster/kubernetes/user-guides/kuberay-gcs-ft.html).
+The similar type of highly available Ray cluster can also be created using API server. The foundation of this
+approach is ensuring high availability Global Control Service (GCS) data. GCS manages cluster-level
+metadata. By default, the GCS lacks fault tolerance as it stores all data in-memory, and a failure can cause the
+entire Ray cluster to fail. To make the GCS fault tolerant, you must have a high-availability Redis. This way,
+in the event of a GCS restart, it retrieves all the data from the Redis instance and resumes its regular
+functioning.
+
+## Creating external Redis cluster
```

Comment:
Perhaps adding a blurb around the use of kind as test cluster or BYOC might be helpfull.